app/pages/returns/ExportedPlasticPackagingWeightPage.scala

```diff
@@ -17,22 +17,41 @@
 package pages.returns
 
 import models.UserAnswers
-import pages.QuestionPage
+import pages._
 import play.api.libs.json.JsPath
 
 import scala.util.Try
 
 case object ExportedPlasticPackagingWeightPage extends QuestionPage[Long] {
 
+  private def exportedAllPlastic(answers: UserAnswers): Boolean = {
+    val manufactured = answers.get(ManufacturedPlasticPackagingWeightPage).getOrElse(0L)
+    val imported     = answers.get(ImportedPlasticPackagingWeightPage).getOrElse(0L)
+    val exported     = answers.get(ExportedPlasticPackagingWeightPage).getOrElse(0L)
+
+    exported >= (manufactured + imported)
+  }
+
   override def path: JsPath = JsPath \ toString
 
   override def toString: String = "exportedPlasticPackagingWeight"
 
   override def cleanup(value: Option[Long], userAnswers: UserAnswers): Try[UserAnswers] =
     value.map(amount =>
```

Comment:
is there any test for the cleanup function?

Reply:
Not yet as I am not sure its staying at this point.

Reply:
ok